### PR TITLE
[t152276] Enhancements: All tags should now close at a next line to avoid <closing></field>. Ordering of resulting tags should now be better by using a custom formatter instead of the bs4 xml formatter. Added a more realistic test xml file and a simpler one for testing attribute ordering

### PR DESCRIPTION
**Pull request by BT-dherreros**
<div> Links to Odoo: </div> <ul>
<li><a target="_blank" href="https://braintec.com/web#view_type=form&model=project.task&id=152276">[t152276] Enhancements to view adapt scripts</a></li>
</ul>

### CODE REVIEW

File: realistic_test.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+Copyright (c) 2024 braintec AG (https://braintec.com)
+All Rights Reserved
+Licensed under the Odoo Proprietary License v1.0 (OPL).
+See LICENSE file for full licensing details.
+-->
```
Comment (BT-anieto) [inline realistic_test.xml]: **[question]** Why this license? Licenses in all files or in none, right?

**Review by BT-anieto — APPROVED** [review]
LGTM, but for this replacement magic, tests are the best reviewers.

File: realistic_test.xml (hunk 1 above)
Comment (BT-dherreros) [reply realistic_test.xml]: I just wanted to have a test file as close to a an actual view file we could have. But yes, maybe having the license there bound us legally to some issues. I could replace it with 2 fake license lines, what do you think about it?

Comment (BT-anieto) [reply realistic_test.xml]: No clue. Maybe HOD has an idea? @BT-rmartin

Comment (BT-rmartin) [reply realistic_test.xml]: @BT-dherreros yes, let's go with 2 fake lines

**Review by BT-cjimeno — APPROVED** [review]
LGTM, I added some changes to try to preserve the order in the attrs too.
I would merge it already, and if we find issues we can solve them later.
Thx for the changes!

**Review by BT-cjimeno — APPROVED** [review]

**Pull request merged**